magprime/configspec.ini

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
And also, it's easy (though not ideal) to hard-code the development.ini for this plugin like we do with core, if we really needed to.  Since only magfest is using this and it's unlikely to contain sensitive data, for now just hardcoding stuff in the .ini is fine.